openedx/features/content_tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,76 @@
+"""
+Tagging API Views
+"""
+from django.http import Http404
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from openedx_tagging.core.tagging.rest_api.v1.serializers import (
+    TaxonomyListQueryParamsSerializer,
+    TaxonomySerializer
+)
+from openedx_tagging.core.tagging.rest_api.v1.permissions import TaxonomyObjectPermissions
+
+from ...api import (
+    create_taxonomy,
+    delete_taxonomy,
+    get_taxonomy,
+    get_taxonomies,
+    update_taxonomy,
+)
+
+
+class TaxonomyView(ModelViewSet):
+    """
+    Docstring
+    """
+
+    serializer_class = TaxonomySerializer
+    lookup_field = "id"
+    permission_classes = [TaxonomyObjectPermissions]
+
+    def get_object(self):
+        """
+        Return the requested taxonomy object, if the user has appropriate
+        permissions.
+        """
+        id = self.kwargs.get("id")
+        taxonomy = get_taxonomy(id)
+        if not taxonomy:
+            raise Http404("Taxonomy not found")
+        self.check_object_permissions(self.request, taxonomy)
+
+        return taxonomy
+
+    def get_queryset(self):
+        """
+        Return a list of taxonomies. If you want the disabled taxonomies, pass enabled=False. If you want all taxonomies (both enabled and disabled), pass enabled=None.
+        """
+        query_params = TaxonomyListQueryParamsSerializer(
+            data=self.request.query_params.dict()
+        )
+        query_params.is_valid(raise_exception=True)
+        enabled = query_params.data.get("enabled", None)
+
+        return get_taxonomies(enabled)
+
```

Comment:
You'll need to investigate how best implement this, but could you also make it so the `list` view accepts an `org` parameter? That `org` can be passed to [`get_taxonomies_for_org`](https://github.com/openedx/edx-platform/pull/32661/files#diff-0ed9e2727b75df5366745749e3b39718ffbefe4b60289d80feab3b0019feb66bR67), or used to limit the returned Taxonomies with [`is_owner`](https://github.com/openedx/edx-platform/pull/32661/files#diff-8a644422ddd022128d13b6ca7c8b672a33d048d1816a80a80491246bdd76f607R53-R64), whatever works best.

Unfortunately, I don't have a way to do this with a Taxonomy queryset filter, sorry.

Reply:
Ok! I will check this tomorrow!

Reply:
Oo.. Can we actually subclass your oel_tagging library's TaxonomyView here? Then all we have to implement is this custom org filtering; the rest of the code can remain the same.

Reply:
Hey @rpenido -- let's leave this PR for next sprint, to do part of https://github.com/openedx/modular-learning/issues/74.
Since your https://github.com/open-craft/openedx-learning/pull/4 PR implements almost everything we need for https://github.com/openedx/modular-learning/issues/73, I think it's fair to split out the org- and content-specific stuff into this separate issue.